project_setup/docker/README.md

```diff
@@ -0,0 +1,264 @@
+# Docker for development
+
+Below we lay out recommendations for using Docker to wrap development
+dependencies. Unfortunately, this is a relatively new space and we don't have
+a consistent, end-to-end setup at the moment. Instead, we'll provide pointers,
+partial solutions and examples. We intend to flesh out complete solutions as
```

Comment:
18F Content Guide recommends using serial commas :)

`partial solutions and examples` -> `partial solutions, and examples.`

ref https://pages.18f.gov/content-guide/punctuation/#commas